pkg/apis/core/types.go

```diff
@@ -3508,6 +3512,11 @@ type LoadBalancerIngress struct {
 	// (typically AWS load-balancers)
 	// +optional
 	Hostname string
+
+	// Ports is a list of error records of service ports
+	// If used every port defined in the service must have an entry in it
```

Comment:
really minor and if it is the only fixup I won't block:

"If used, every port defined in the service must have an entry" - the comma make it easier to read and "in it" is not needed

Reply:
Thank you, fixed.